tasks/run.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var exec = require('../lib/exec');
+var spawn = require('child_process').spawn;
+var packageJson = require(process.cwd() + '/package.json');
+var normalizeName = require('../lib/normalize-name');
+var LOCAL_PORT = 3002;
```

Comment:
ok, I wonder if that's going to be inconvenient. If production is set by `export PORT=1234` I think localhost should also be.